README.rst

```diff
@@ -73,6 +73,20 @@ Some Python HTTP clients that you can use to retrieve HTML include `requests`_,
 .. _urllib.request module: https://docs.python.org/3/library/urllib.request.html
 
 
+Test data
+---------
+
+As a precondition for inclusion in this library, each recipe scraper is required to include test coverage to demonstrate that it can accurately retrieve recipe information from the website it relates to.
+
+To achieve that, the development source repository includes a minimal number of test HTML files from the public web -- often only one or two per recipe website -- to confirm the scraper's behaviour.  These test files are not included in the [published releases](https://pypi.org/project/recipe-scrapers/) of the library.
+
+If you are the author/publisher of any of those files and would prefer for them to be removed, there are at least two ways for you to do that:
+
+  * You may follow GitHub's [DMCA notice and takedown process](https://docs.github.com/en/site-policy/content-removal-policies/dmca-takedown-policy) that is available to you as a copyright owner.
```

Comment:
Let's instead provide a reference to GitHub's more-general content-removal request processes: https://docs.github.com/en/site-policy/content-removal-policies/submitting-content-removal-requests